modules/shell/command_test.go

```diff
@@ -33,9 +33,13 @@ func TestRunCommandAndGetOutputOrder(t *testing.T) {
 	bashCode := fmt.Sprintf(`
 echo_stderr(){
 	(>&2 echo "%s")
+	# Add sleep to stabilize the test
+	sleep .01s
 }
 echo_stdout(){
 	echo "%s"
+	# Add sleep to stabilize the test
+	sleep .01s
```

Comment:
This test was getting unstable due to wrong interleaving from timing issues, so added a delay to make this more consistent.

Reply:
Could you elaborate? I still dont understand what was causing the instability